pupil-spa/src/app/app.module.ts

```diff
@@ -40,6 +40,7 @@ import { WarmupIntroComponent } from './warmup-intro/warmup-intro.component';
 import { WarmupLoadingComponent } from './warmup-loading/warmup-loading.component';
 import { WarmupQuestionService } from './services/question/warmup-question.service';
 import { WindowRefService } from './services/window-ref/window-ref.service';
+import { AppCountService } from './services/app-count/app-count.service';
```

Comment:
It's counting Checks, not Apps.   I would go with `CheckCountService` or `AppUsageService`.   `AppUsageService` works for me because we may extend this to store other wonderful variables like the time and date the app was initialised, the number of times the network dropped out, or whatever.

Reply:
@jon-shipley renamed